plc4j/pom.xml

```diff
@@ -45,7 +45,37 @@
 
   <build>
     <plugins>
-
+        <plugin>
```

Comment:
Hey @cschneider there is another invocation of the maven-bundle-plugin (L:200). Is this okay or hould this be removed?

Reply:
And I would also add the same part (with the part from L:200 with maven-bdunle-plugin and jar plugin???) to protocols/pom.xml, right?

Reply:
Oh .. sorry I missed that it was already present. So yes the other invocation should be removed. Actually to make the diff look cleaner I will merge my changes into the existing config.

Reply:
It also makes sense to add the same bundling config to protocol. I can update the PR.

Reply:
Thank you : )
Then I will check how the bundles look like and if some of them already work (one can hope...).